### PR TITLE
docs: added login page

### PR DESCRIPTION
**Pull request by SinchanaMS**
Created the Login Page.

image

### CODE REVIEW

File: stylesheets/login-page.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
+.login-container {
+    background: var(--CARD-BG);
+    min-width: max-content;
+    width: fit-content;
+    border-radius: 5px;
+    border: none;
+    margin: 5rem auto;
+    padding: 3rem;
+
```
Comment (j24m) [inline stylesheets/login-page.css:9]: one extra spacing.

**Review by j24m — APPROVED** [review]
hello, 5 stars for small PR, good code , good class_names, and beautiful UI : )

File: docs/login-page.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            <div class="auth-opts">
+                <div class="checkbox-btn remember-me">
+                    <input type="checkbox" name="checkbox">
+                    <label class="label">Remember Me</label>
+                </div>
+                <a href="#" class="forgot-pwd">Forgot Password?</a>
+            </div>
+            <button class="btn link-btn-outline login-btn icon-dark">
+                <a href="#" class="link-in-btn p-lg">Login</a>
+            </button>
```
Comment (PuneetP16) [inline docs/login-page.html:75]: Suggestion: Use required attribute and  wrap up in html form tag to run checks if password and email fields are not empty on submit button click.

**Review by PuneetP16 — APPROVED** [review]
Looks good to merge.

File: docs/login-page.html (hunk 2 above)
Comment (SinchanaMS) [reply docs/login-page.html:75]: Thank you! I'll do that.

**Pull request merged**